### ISSUE TITLE
Icon for desktop

### ISSUE THREAD

**Issue by jenokizm** [cthedot/icongen#1]
Hi, cool app! Please add create .ico for desktop programm (sample Windows 7)

**Comment by cthedot** [comment]
Do you have links about how such an icon file is build? Generating PNGs is quite simple, ICO may be more difficult though...
Thanks for the idea!

**Comment by jenokizm** [comment]
No, I unfortunately do not know exactly how to keep the .ico images, but I know that this "container" which should contain the options for different size and bit depth, and number of colors.
I use programm IconWorkshop, but not support vector(
![ashampoo_snap_2016 11 26_19h00m09s_001_](https://cloud.githubusercontent.com/assets/11296510/20641525/1044348e-b40b-11e6-99ad-0c1712edeb6d.png)
so I have to first save the icon for each size and then manually import it. I would be very time if your service is automatically reduced or zoomed vector and created the .ico with a full container.
